include/mbgl/style/expression/format_section_override.hpp

```diff
@@ -30,14 +28,15 @@ class FormatSectionOverride final : public Expression {
 
         return defaultValue.match(
             [&context](const style::PropertyExpression<T>& e) { return e.getExpression().evaluate(context); },
-            [](const T& t) -> EvaluationResult { return t; });
+            [](const T& t) noexcept -> EvaluationResult { return t; });
     }
 
     void eachChild(const std::function<void(const Expression&)>& fn) const final {
-        defaultValue.match([&fn](const style::PropertyExpression<T>& e) { fn(e.getExpression()); }, [](const T&) {});
+        defaultValue.match([&fn](const style::PropertyExpression<T>& e) { fn(e.getExpression()); },
+                           [](const T&) noexcept {});
     }
 
-    bool operator==(const Expression& e) const final {
+    bool operator==(const Expression& e) const noexcept final {
```

Comment:
```suggestion
    bool operator==(const Expression& e) const final {
```

`defaultValue.match` is not `noexcept`.